src/hb-buffer-serialize.cc

```diff
@@ -128,49 +128,54 @@ _hb_buffer_serialize_glyphs_json (hb_buffer_t *buffer,
 
     *p++ = '{';
 
-    APPEND ("\"g\":");
-    if (!(flags & HB_BUFFER_SERIALIZE_FLAG_NO_GLYPH_NAMES))
-    {
-      char g[128];
-      hb_font_glyph_to_string (font, info[i].codepoint, g, sizeof (g));
-      *p++ = '"';
-      for (char *q = g; *q; q++) {
-	if (*q == '"')
-	  *p++ = '\\';
-	*p++ = *q;
-      }
-      *p++ = '"';
-    }
-    else
```

Comment:
I’m not sure if `_serialize_glyphs()` functions are the right place to this modification, since the API is clearly named and expects a glyph buffer. May be we need a new `_serialize_unicode()` API.

Reply:
Hmm. This is a good point. But if we're going that way, it probably also means we need a `hb_buffer_serialize` to switch between the two.